adls-plugins/docs/ADLSBatchSink-batchsink.md

```diff
@@ -25,10 +38,36 @@ needed for the distributed file system. (Macro-enabled)
 
 **textDelimiter:** Delimiter to place between fields. Only used by the text output format. Defaults to tab.
 
+DevNote
```

Comment:
User guide can’t have dev notes. Move it to readme.

Reply:
DevNote was an incorrect title, these are actually UserNotes to help people understand scenarios to use KeyVault option securely. I have renamed the title accordingly (`Notes on using KeyVault approach`). Let me know if you still feel it should be moved to Readme?